src/main/scala/org/broadinstitute/dsde/rawls/dataaccess/HttpGoogleServicesDAO.scala

```diff
@@ -90,7 +90,9 @@ class HttpGoogleServicesDAO(
     }
   }
 
-  override def setupWorkspace(userInfo: UserInfo, projectId: String, workspaceId: String, workspaceName: WorkspaceName): Future[Unit] = {
+  private def getBucketName(workspaceId: String) = s"${groupsPrefix}-${workspaceId}"
+
+  override def setupWorkspace(userInfo: UserInfo, projectId: String, workspaceId: String, workspaceName: WorkspaceName): Future[GoogleWorkspaceInfo] = {
```

Comment:
this signature has been changed to return the name of the bucket and the groups created wrapped in GoogleWorkspaceInfo